lib/globals.d.ts

```diff
@@ -1,4 +1,22 @@
+// This will clash with the node 'global' namespace...
```

Comment:
This does not require any immediate action but the tsc command will complain about global namespace being declared twice.